### PR TITLE
Code coverage

### PR DESCRIPTION
**Pull request by fenollp**
What it brings: https://coveralls.io/github/fenollp/kazoo
We have to activate https://coveralls.io/github/2600hz/kazoo though. It is totally free even though they try to make it look like you need to register a credit card.

We're at 5.6% coverage with this test suite :/

### CODE REVIEW

File: make/deps.mk [hunk 1]
```diff
@@ -6,6 +6,11 @@ DEPS = lager eiconv gen_smtp amqp_client cowboy jesse jiffy certifi couchbeam ws
 BUILD_DEPS = parse_trans
 IGNORE_DEPS = hamcrest
 
+ifeq ($(USER),travis)
```
Comment (fenollp) [inline make/deps.mk]: It is travis-only though. Not sure what an env var would change here

File: .travis.yml [hunk 2]
```diff
@@ -33,3 +33,6 @@ script:
   - make eunit
   - # ERL_LIBS="$HOME/proper" ERLC_OPTS='-DPROPER' make compile-test
   - # ERL_LIBS="$HOME/proper" make eunit
+
+after_success:
+  - make coverage-report
```
Comment (skorobkov) [inline .travis.yml]: It is cause a build fail on travis-ci if organisation dont use this service
https://travis-ci.org/dinkor/kazoo/jobs/170391627

Comment (fenollp) [inline .travis.yml]: Should be fixed in https://github.com/2600hz/kazoo/pull/2735 ;)

Comment (skorobkov) [inline .travis.yml]: Yes. It is ok now. Thank you.

**Pull request merged**